config/v1/types_cluster_version.go

```diff
@@ -780,11 +806,24 @@ type ConditionalUpdate struct {
 	// +required
 	Release Release `json:"release"`
 
+	// riskNames represents the set of the names of conditionalUpdateRisks in the status that are exposed to the release in this conditional update.
+	// A condition update is accepted only if each of its risk is either not applied to the cluster or considered acceptable by the cluster administrator.
+	// Entries must be unique and must not exceed 256 characters.
+	// riskNames must not contain more than 500 entries.
+	// +openshift:enable:FeatureGate=ClusterUpdateAcceptedRisks
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:items:MaxLength=256
+	// +kubebuilder:validation:MaxItems=500
+	// +listType=set
+	// +optional
+	RiskNames []string `json:"riskNames,omitempty"`
+
 	// risks represents the range of issues associated with
 	// updating to the target release. The cluster-version
 	// operator will evaluate all entries, and only recommend the
 	// update if there is at least one entry and all entries
 	// recommend the update.
+	// Deprecated: the risks has been deprecated by riskNames.
```

Comment:
Something worth noting - this shows up on the non-featuregated version of the API.

I'm not sure there is much of a workaround for this unless you wait to add this deprecation text until you actually go to promote the feature-gate that deprecates this to GA.

I would probably recommend removing this for now.

Reply:
Removed.